requirements.txt

```diff
@@ -5,7 +5,6 @@ funcsigs
 functools32
 future
 futures                 # required by trollius
```

Comment:
There are a number of packages that can be removed after moving to Python 3:
- `futures`, `functools32`, `subprocess32`, `ipaddress` (all built in to Python 3)
- `trollius` (not needed by spead2 on Python 3)
- possibly `backports-abc`, `backports.ssl-match-hostname`, although I haven't tested that (it's possible that some other package will depend on them).
- Probably `unittest2`, `linecache2`, `traceback2`, `singledispatch`, but again, some packages might depend on them.

Reply:
It seems to build if you remove linecache2, traceback2, singledispatch, backports-abc, backports.ssl-match-hostname, unittest2. I haven't tried running it though.